github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
+
 def _print_pr(pr, **args):
     if args['numberonly']:
         print "%d" % pr.number
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title)
+        commit_status = _return_status(pr.get_commits())
+        print "#%d [%s] %10s <- %-30s    %s - Commit/Build Status:%s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title, commit_status)
```

Comment:
Yeah i guess its not the right output, but i also dont think that it will be clear if we just have the status there. People will be confused as to what its the status of, the PR, the commit, the mergeability. I think the tabulate is an option but want to figure out what we want the header of that column to say.